### ISSUE TITLE
 gopsutil/v3 windows load  is not supported

### ISSUE THREAD

**Issue by sanxun0325** [shirou/gopsutil#1014]
**Describe the bug**
[A clear and concise description of what the bug is.]

**To Reproduce**
```go
// paste example code reproducing the bug you are reporting
```
go.mod

github.com/shirou/gopsutil/v3 v3.20.11

windows load not implemented yet

**Expected behavior**
Directly introducing github.com/shirou/gopsutil v3.20.11  windows Load is supported

**Environment (please complete the following information):**
 - [x] Windows: [paste the result of `ver`]
 - [ ] Linux: [paste contents of `/etc/os-release` and the result of `uname -a`]
 - [ ] Mac OS: [paste the result of `sw_vers` and `uname -a`
 - [ ] FreeBSD: [paste the result of `freebsd-version -k -r -u` and  `uname -a`]
 - [ ] OpenBSD: [paste the result of `uname -a`]

**Additional context**
[Cross-compiling? Paste the command you are using to cross-compile and the result of the corresponding `go env`]

**Comment by Lomanic** [comment]
I don't understand your issue and you weren't bothered enough to fill the template. `load.Avg()` is implemented on windows on both v2 and v3 (#977 and #998).

**Comment by sanxun0325** [comment]
> I don't understand your issue and you weren't bothered enough to fill the template. `load.Avg()` is implemented on windows on both v2 and v3 (#977 and #998).
Yes, I've seen support for load.Avg() in v3. Maybe I should ask when the next release will be issued.

**Comment by Lomanic** [comment]
OK, indeed, [gopsutil is released each month](https://github.com/shirou/gopsutil#tag-semantics) so it will be released in v3.20.12 at the end of the month.

**Comment by sanxun0325** [comment]
> OK, indeed, [gopsutil is released each month](https://github.com/shirou/gopsutil#tag-semantics) so it will be released in v3.20.12 at the end of the month.

Thank you for answering.
